test/synapse/2019-06-01/azartifacts/zz_generated_constants.go

```diff
@@ -130,31 +130,31 @@ func (c BlobEventType) ToPtr() *BlobEventType {
 type CassandraSourceReadConsistencyLevels string
 
 const (
-	CassandraSourceReadConsistencyLevelsAll         CassandraSourceReadConsistencyLevels = "ALL"
+	CassandraSourceReadConsistencyLevelsALL         CassandraSourceReadConsistencyLevels = "ALL"
```

Comment:
This and the below constants are a bit weird to me as to why they're capitalized

Reply:
This is from the [swagger](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/Pipeline.json#L2023).